main.go

```diff
@@ -272,6 +275,11 @@ func runCLIApp(c *cli.Context) (err error) {
 		return fmt.Errorf("parsing flags failed: %w", err)
 	}
 
+	mountOptions, err := config.ReadConfigFile(flags.ConfigFile)
```

Comment:
We can use  `mountConfig` instead of `mountOptions` like other places?

Reply:
done